sr_robot_launch/launch/sr_right_ur10arm_hand.launch

```diff
@@ -22,6 +22,8 @@
   <arg name="eth_port" default="$(optenv ETHERCAT_PORT eth0)"/>
   <!-- Set whether to load a scene to restrain the movement of the robot -->
   <arg name="scene" default="false"/>
+  <!-- Set to true if you want the robot to start in the "home" positions -->
+  <arg name="start_home" default="false"/>
```

Comment:
we should add the home position values here so we know where to change them

Reply:
Added!